tabbycat/notifications/utils.py

```diff
@@ -108,6 +112,7 @@ def _create_ballot(result, scoresheet):
         return ballot
 
     if isinstance(results, VotingDebateResult):
+        results.load_scoresheets()
```

Comment:
So the problem seemed to be that the scoresheets were not being loaded. Any idea how to fix it?